src/broker.js

```diff
@@ -19,16 +19,18 @@ function Broker(provider, brokerUrl, tags, username, password) {
 	this._options.tags = tags;
 	this._options.username = username;
 	this._options.password = password;
+	this._requestOptions = (this._options.username && this._options.password) ? { 'auth': { 'user': this._options.username, 'password': this._options.password } } : {};
 };
 
 // Find Pacts returns the raw response from the HAL resource
 Broker.prototype.findPacts = function (tag) {
-	logger.debug("finding pacts: ", tag)
-	var linkName = (tag) ? 'pb:latest-provider-pacts-with-tag' : 'pb:latest-provider-pacts';
+	logger.debug("finding pacts for Provider:", this._options.provider, ", Tag:", tag)
+
+	var linkName = tag ? 'pb:latest-provider-pacts-with-tag' : 'pb:latest-provider-pacts';
```

Comment:
Could be `var linkName = 'pb:latest-provider-pacts' + tag ? '-with-tags' : '';`. Might not even need to save it to a variable and just use it inline.